data/units.yaml

```diff
@@ -0,0 +1,6 @@
+gravimetric:
+  label: Gravimetric Units
+  unit: ms^-2
+euros:
+  label: Euros
+  unit: €
```

Comment:
I'd add the `type` and the `kind` like below.
For money I'd prefer to use `USD`, `EUR` rather than a symbol for consistency (not all the currencies have a well know or easy-to-type symbol)

```yaml
gravimetric:
  kind: acceleration
  label: Gravimetric Units
  type: string
  unit: ms^-2
euros:
  kind: currency
  label: Euros
  type: string
  unit: EUR
```

Reply:
Done a quick bit of reading on this. For this particular data the gravimetric units are actually measured in µg m^-3, so not an acceleration. That's no problem to update here for this project, but highlights some considerations for a wider standardised schema/unit definitions:
- Just because a unit has a common name, we shouldn't assume that it is standardised across fields. E.g. a unit of "irradiance" may have totally different units depending on whether you are talking about an experiment involving lasers vs an astrophysics study
- Units can be expressed with different magnitudes. Gravimetric units provide a simple example as here they are expressed in µg, but are often referred to in terms of g.